src/script/script.cpp

```diff
@@ -306,3 +311,77 @@ bool GetScriptOp(CScriptBase::const_iterator& pc, CScriptBase::const_iterator en
     opcodeRet = static_cast<opcodetype>(opcode);
     return true;
 }
+
+bool CScriptNum::IsMinimallyEncoded(const std::vector<uint8_t> &vch,
+                                    const size_t nMaxNumSize) {
```

Comment:
We normally have this on one line, I'd probably want to keep it that way, also { on new line